src/components/TheSidebar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <button class="collapse-icon" @click="toggleSidebar">
+    <i class="fa-solid fa-bars"></i>
+  </button>
+
+  <div class="sidebar" :class="{ collapsed: collapsed }">
+    <div class="sidebarlink">
```

Comment:
You may want to change the class name into "sidebar_link" or "sidebarLInk" depending on what convention you follow.